package.json

```diff
@@ -16,20 +16,25 @@
     "async": "^0.9.0",
     "compression": "^1.5.1",
     "debug": "^2.1.1",
+    "ejs": "^2.3.3",
     "errorhandler": "^1.4.1",
-    "loopback": "^2.0.0",
-    "loopback-boot": "^2.0.0",
+    "loopback": "^2.19.2",
+    "loopback-boot": "^2.8.11",
+    "loopback-datasource-juggler": "^2.33.6",
     "mkdirp": "^0.5.0",
     "posix-getopt": "^1.1.0",
+    "strong-control-channel": "^2.0.27",
```

Comment:
That said, I'm not even sure what this dependency was added for. It doesn't seem to be used.